vendor/assets/stylesheets/uswds_overrides/components/_footer.scss

```diff
@@ -1,15 +1,143 @@
-.usa-footer-big {
-  .usa-footer-primary-link {
-    background-image: image-url('uswds/arrow-down.png');
-    background-image: image-url('uswds/arrow-down.svg');
+.site-footer{
+  html, body , div, span, applet, object, iframe,
```

Comment:
I'm thinking I'll do this in each of the sections until we actually change some of the defaults. For example, this site shouldn't use white as the background color; it should use the dark blue.